backend/src/usersAvatar/controllers/usersAvatar-controller.ts

```diff
@@ -0,0 +1,13 @@
+import { Controller, Get, Param } from "@nestjs/common";
+import { UsersAvatarService } from "../services/usersAvatar.service";
+@Controller("usersAvatar")
+class UsersAvatarController {
+	constructor(private memeService: UsersAvatarService) {}
+
+	@Get("local/:name")
```

Comment:
Change url name

Reply:
This path should be: "/api/userAvatar/:soruce" IMO